### DISCUSSION TITLE
can a visual highlight be added to vim.sneak?

### DISCUSSION THREAD

**Discussion by arkosno** [VSCodeVim/Vim#8392] [Ideas]
visual functionally where the inputted character is highlighted just like leap.nvim.

https://github.com/ggandor/leap.nvim

I feel that this is way better than easymotion

**Comment by darianmorat** [comment]
I would recommend you [THIS](https://marketplace.visualstudio.com/items?itemName=szaldivar.szaldivar-leap) extension
